python/cog/server/webhook.py

```diff
@@ -47,11 +49,18 @@ def webhook_caller(webhook: str) -> Callable[[Any], None]:
     def caller(response: PredictionResponse) -> None:
         if throttler.should_send_response(response):
             if PYDANTIC_V2:
-                dict_response = jsonable_encoder(
-                    response.model_dump(exclude_unset=True)
-                )
+                response_object = response.model_dump(exclude_unset=True)
             else:
-                dict_response = jsonable_encoder(response.dict(exclude_unset=True))
+                response_object = response.dict(exclude_unset=True)
+
+            if "output" in response_object:
+                response_object["output"] = upload_files(
+                    response_object["output"],
+                    upload_file=upload_file,  # type: ignore
+                )
```

Comment:
What was the thinking of doing this here in the webhook handler rather than in the worker where we currently do the file uploads?

https://github.com/replicate/cog/blob/018aca50f9d9d43ff784f1ea239c7cede72dc7c0/python/cog/server/runner.py#L452-L453

Reply:
Had a chat with @aron and we will move this to the runner and see in the CI what breaks in terms of tests, updated in: [8707aba](https://github.com/replicate/cog/pull/2185/commits/8707abac640c5bff663512c0ad7134408488e4ee)